src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const summing = (number1) =>
```

Comment:
```suggestion
  const getSum = (number1) =>
```
function/method name should always start with verb